backend/api/tests/views/test_views_datasample.py

```diff
@@ -223,7 +212,8 @@ def test_data_create_upload(self):
             "json": json.dumps(
                 {
                     "data_manager_keys": [self.data_manager_key],
-                }
+                },
+                cls=DjangoJSONEncoder,
```

Comment:
Was this not needed before, or is it an unrelated improvement?

Reply:
It was not needed before, as we had before `self.data_manager_key` and `self.data_manager_key_uuid` (the formest being a string). I decided to let use the encoder we used in our REST API instead of manually converting it, but I am open to revert to the previosu behavior!

Reply:
It seems like a good idea not to store the same information twice indeed 👍